src/Umbraco.Web/Models/ContentEditing/Tab.cs

```diff
@@ -23,5 +23,8 @@ public class Tab<T>
 
         [DataMember(Name = "properties")]
         public IEnumerable<T> Properties { get; set; }
+
+        [DataMember(Name = "cssClass")]
+        public string CssClass { get; set; }

```

Comment:
Unfortunately this breaks the auto mapper unit test because this is a non-mapped property during mapping operations. An explicit Ignore needs to be added in the Umbraco.Core.Models.PropertyGroup -> Umbraco.Web.Models.ContentEditing.Tab mapping

Reply:
Fixed here https://github.com/umbraco/Umbraco-CMS/commit/ffd08d0a2ef259f0772da32577d4a7b7232da75b